README.md

```diff
@@ -209,6 +209,24 @@ try {
 }
 ```
 
+### Renew Authentication
+
+Creates a new request to renew the authentication and get fresh new credentials using a valid Refresh Token.
```

Comment:
can I create something "old" ? xD just kidding, but the term "(re)new" is repeated several times, maybe we can remove the first